interface/src/components/grid.js

```diff
@@ -24,29 +31,33 @@ class Grid extends Component{
                 if(!label.includes(data[i][type])){
                     label.push(data[i][type])
                 }
-            }
+            }          
+
             color= getRandomColors(label.length)
             let plots = new Array();
             let xpos = 1; //starting xpos and ypos at 1 so the stroke will show when we make the grid below
             let ypos = 1;
             let width = 25;
             let height = 25;
             let gcol='';
-            let x,y,idx,name='';
+            let x,y,idx,text,name='';
+            let list=[];
             // iterate for rows 
             for (let row = 0; row < mrow; row++) {
                 plots.push( new Array() );
-                
+                name= '';
                 // iterate for cells/columns inside rows
                 for (let column = 0; column < mcol; column++) {
-                    gcol="#fff"
-                    data.forEach((set)=>{
-                        name= set.experiments;
+                    gcol= "#fff";
```

Comment:
Use full 6 character color value for these: eg: `#ffffff`

Reply:
Will do so in the next commit!